Sources/Core/GTMSessionFetcher.m

```diff
@@ -872,6 +872,26 @@ - (void)beginFetchMayDelay:(BOOL)mayDelay
     }
   }
 
+  BOOL stopped;
```

Comment:
This code is basically a subset of `-startDelayState:`, it would work to reuse that with the state being passed being `kDelayStateNotDelayed`, but I wasn't sure if that would be a little too cryptic. We could comment it to make it clear the call is being done for code reuse alone.  Thoughts?

Reply:
I think it's better to leave it out here explicitly. This code is complicated enough that keeping it here makes it easier for me to follow than nesting it in.